etna/lib/etna/application.rb

```diff
@@ -80,7 +87,7 @@ def run_command(config, cmd = :help, *args)
     cmd = cmd.to_sym
     if commands.key?(cmd)
       commands[cmd].setup(config)
-      commands[cmd].execute(*args)
+      commands[cmd].execute(*commands[cmd].fill_in_missing_params(args))
```

Comment:
Some etna commands take a lot of parameters.   With this change, if you forget to fill out all parameters, it will prompt you for the missing params one at a time.  Makes it super easy to use commands from the CLI.